packages/agoric-cli/src/lib/keys.js

```diff
@@ -0,0 +1,14 @@
+// @ts-check
+import { normalizeBech32 } from '@cosmjs/encoding';
+import { execSync } from 'child_process';
+
+export const normalizeAddress = literalOrName => {
+  try {
+    return normalizeBech32(literalOrName);
+  } catch (_) {
+    // not an address so try as a key
+    const buff = execSync(`agd keys show --address ${literalOrName}`);
```

Comment:
exported ambient authority. please add a comment to acknowledge the technical debt.